### ISSUE TITLE
Need an efficiency slider

### ISSUE THREAD

**Issue by samdave123** [RainbowMiner/RainbowMiner#2065]
ATM we're experiencing a combination of higher ambient temperatures, higher power cost, and lower profitability. Efficiency is becoming increasingly important.

The formulae for the heat control slider reduces heat by increasing power cost and minimising power usage. I don't want to minimise power usage, I want to maximise efficiency. The slider doesn't quite achieve this, because it can favour whatever algorithms are lower power rather than which are most efficient.

**Comment by RainbowMiner** [comment]
I am not sure. How would you describe efficiency in this context, exactly?

**Comment by samdave123** [comment]
Maximum sat/W. I want to configure the profit-switcher to use whatever is highest sat/W and right now there's no way to do this.

**Comment by RainbowMiner** [comment]
Ah, ok! So you want to disable the effect of the power cost on the income for the benefit of a price comparison in sat/W, right?

**Comment by samdave123** [comment]
Yes precisely. You explain it so much more concisely and simply than I do.

**Comment by RainbowMiner** [comment]
I think, that's actually a very interesting enhancement. I am now evaluating, if it can be combined with the current price biasing strategies, or if the latter need to be revised.
My gut feeling tells me, it can be easily cascaded - that would give us total control, actually.

**Comment by samdave123** [comment]
It would be neat to have a slider in the webui with a weighting. Either 100% efficiency and 0% profitability, 75%/25%, 50%/50%, 25%/75%, or just normal profit-switching.

This should work much better for the desired effect of switching against the real cost of making me sweat buckets in my flat in the summer.

**Comment by samdave123** [comment]
This would 100% work better than using heat control values under 2.0.
